tests/test_stac_core.py

```
@@ -0,0 +1,53 @@
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)